go.mod

```diff
@@ -7,7 +7,6 @@ require (
 	github.com/Workiva/go-datastructures v1.0.50 // indirect
 	github.com/aergoio/aergo-actor v0.0.0-20190219030625-562037d5fec7
 	github.com/aergoio/aergo-lib v1.1.0
-	github.com/aergoio/badger v1.6.0-gcfix // indirect
```

Comment:
It removed reference to `badger`. Is this OK?
Above this line there is reference to `aergo-lib` that uses `badger` too